app/src/main/java/com/hamdan/acceleromatorminiproject/view/compose/MainPage.kt

```diff
@@ -15,6 +16,7 @@ fun MainPageContent(
     state: SensorViewModel.SensorState,
     onReadyClicked: () -> Unit,
     onErrorDismiss: () -> Unit,
+    toHighScores: () -> Unit,
```

Comment:
Shouldn't this be onHighScoreButtonPressed or something along those lines?

Reply:
yes, adjusted to follow convention of lambdas in project already